core/unit_test/TestViewAPI.hpp

```diff
@@ -1033,25 +1033,25 @@ class TestViewAPI {
     hView3 hv_3("dView3::HostMirror", N0);
     hView4 hv_4("dView4::HostMirror", N0);
 
-    dView0 dv_0_1(nullptr, 0);
+    dView0 dv_0_1(nullptr);
```

Comment:
Why did you change?

Reply:
Because this is failing with these changes
```
Constructor for Kokkos::View 'UNMANAGED' has mismatched number of arguments. The number of arguments = 1 neither matches the dynamic rank = 0 nor the total rank = 0
2: 
2: Backtrace:
2:  0 KokkosCore_UnitTest_Serial2 0x00000001016d3fd8                                                                                                                                                                                                                                                                                                                                                     Kokkos::Impl::save_stacktrace() +   24
2:  1 KokkosCore_UnitTest_Serial2 0x00000001016cce24                                                                                                                                                                                                                                                                                     Kokkos::Impl::traceback_callstack(std::__1::basic_ostream<char, std::__1::char_traits<char> >&) +   36
2:  2 KokkosCore_UnitTest_Serial2 0x00000001016ccee0                                                                                                                                                                                                                                                                                                                                               Kokkos::Impl::host_abort(char const*) +   40
2:  3 KokkosCore_UnitTest_Serial2 0x0000000100fec074                                                                                                                                                                                                                                                                                                                                                              __clang_call_terminate +    0
2:  4 KokkosCore_UnitTest_Serial2 0x00000001010db258 void Kokkos::Impl::runtime_check_rank<Kokkos::View<double, Kokkos::Serial> >(Kokkos::View<double, Kokkos::Serial> const&, unsigned long, unsigned long, bool, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, unsigned long, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) + 1292
2:  5 KokkosCore_UnitTest_Serial2 0x00000001010b5db8                                                                                                                                                                                                                                                                                                       Test::TestViewAPI<double, Kokkos::Serial>::run_test_contruction_from_layout() +  736
2:  6 KokkosCore_UnitTest_Serial2 0x00000001010b5094                                                                                                                                                                                                                                                                                                                                            Test::serial_view_api_b_Test::TestBody() +   24
2:  7 KokkosCore_UnitTest_Serial2 0x0000000101691864                                                                                                                                                                                                                                            void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) +  100
2:  8 KokkosCore_UnitTest_Serial2 0x00000001016917b0                                                                                                                                                                                                                                                                                                                                                                testing::Test::Run() +  452
2:  9 KokkosCore_UnitTest_Serial2 0x0000000101692e44                                                                                                                                                                                                                                                                                                                                                            testing::TestInfo::Run() +  480
2: 10 KokkosCore_UnitTest_Serial2 0x0000000101693880                                                                                                                                                                                                                                                                                                                                                           testing::TestSuite::Run() +  356
2: 11 KokkosCore_UnitTest_Serial2 0x00000001016a45b8                                                                                                                                                                                                                                                                                                                                      testing::internal::UnitTestImpl::RunAllTests() + 2144
2: 12 KokkosCore_UnitTest_Serial2 0x00000001016a3bb4                                                                                                                                                                                      bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) +  100
2: 13 KokkosCore_UnitTest_Serial2 0x00000001016a3b1c                                                                                                                                                                                                                                                                                                                                                            testing::UnitTest::Run() +  124
2: 14 KokkosCore_UnitTest_Serial2 0x0000000100feb818                                                                                                                                                                                                                                                                                                                                                                                main +   52
2: 15                        dyld 0x000000010212d08c                                                                                                                                                                                                                                                                                                                                                                               start +  520
1/1 Test #2: KokkosCore_UnitTest_Serial2 ......Subprocess aborted***Exception:   9.32 sec
```

Reply:
OK.  I saw the definition of the dViewX type now.  Makes sense.